### DISCUSSION TITLE
Apiman 3 is out!

### DISCUSSION THREAD

**Discussion by msavy** [apiman/apiman#2365] [Announcements]
![image](https://user-images.githubusercontent.com/423513/204885474-f56f4540-6de8-42c0-8a9b-6ef3bd37f6f0.png)

##  Apiman 3: A milestone

I'm delighted to say that Apiman 3 is out. It's been a massive amount of hard work and long hours, and I'm looking forward to getting your feedback!

Please check out my [Apiman 3 blog to mark the release](https://www.apiman.io/blog/apiman-3.0.0.Final/), and you can try out the new website I've spent the last few weeks developing from the ground-up (automation should now be much easier).

Let me know in this thread what you think!

## The future

A lot of work has gone on behind the scenes in Apiman 3 to make Apiman ready for rapid development again. I would like to send a special 'thank you' to all those who have worked with me to make this happen, especially the fine people over at Scheer PAS who worked with me extensively on Apiman 3. 

As with many open source projects, a large number of companies are benefit from the hard work put into Apiman, but few contribute back. Please [consider working with Apiman's developers for your support, development, and integration needs](https://www.apiman.io/support.html), as this helps ensure full-time developers can continue working on a project that so many benefit from — plus, you won't find anyone else who knows Apiman so well 😊! 

I'd like to give special thanks to @julian-eins, @volkflo, @BastianGem, and Olli (in no particular order) for their excellent work on Apiman's new developer portal Angular app. If you're thinking of contributing to Apiman, that might be a good place to start — https://github.com/apiman/apiman-developer-portal

## Download now

We have a new docker compose distro that makes it easy to try Apiman!

Head over to the Apiman website or the GitHub release section – https://www.apiman.io/download.html

**Comment by umeshtiwari23** [comment]
Does it supports canary testing between APIv1 & APIv2? e.g. 70% of request goes to APIv1 and 30% goes to APIv2

**Reply by msavy** [reply]
If I understand the use-case correctly you'd probably need to write a custom plugin. 

Please open an issue to explain what you want in more depth and we can investigate a bit m ore.

https://github.com/apiman/apiman/issues/new?assignees=&labels=T%3A+Feature+Request%2CS%3A+Triage&template=feature_request.yml

**Reply by msavy** [reply]
See: https://github.com/apiman/apiman/issues/2393
